src/coll/mod.rs

```diff
@@ -43,10 +45,13 @@ impl Collection {
         db: Database,
         name: &str,
         create: bool,
+        max_time_ms: Option<i64>,
```

Comment:
In retrospect,everything beyond the `Database` and Name should probably be abstracted to an options struct (and ditto for creating a database only needing `Client` and name). Are you interested in taking a stab at this, or would you prefer that I make the change first and then you can rebase it?